less/home-page.less

```diff
@@ -1,32 +1,89 @@
-.home {
+// testing a commit ignore this comment
+.content-container {
+  @media @desktop {
+    width: 85%;
+  }
+}
 
-  .content-section {
-    margin: 30px 0;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
+h2 {
+  @media @desktop {
+    font-size: 2.5rem;
+  }
+}
+
+h4 {
+  @media @desktop {
+    font-size: 2rem;
+  }
+}
+
+p {
+  @media @desktop {
+    font-size: 1.3rem;
+  }
 
-   
-    
+}
+header {
+  padding-top: 1rem;
+  .border {
+    border-top: 2px dashed gray;
+    width: 100vw;
+    position: absolute;
+    left: 0;
+  }
+
+  img {
+    margin-bottom: 1.7rem;
+  }
+}
+
+.home {
+  border-top: 2px dashed gray;
+  margin-top: 1rem;
+  .flexy(flex, column, null, center, center);
+  .content-section {
```

Comment:
You could use one of the colors already defined in the `variables.less` file to match the design file color scheme more closely for the dashed border.